src/components/VmActions/index.js

```diff
@@ -279,8 +280,21 @@ class VmActions extends React.Component {
         confirm={{ title: msg.yes(), onClick: () => onRemove({ preserveDisks: this.state.removePreserveDisks }) }}
       />)
 
-    return (
-      <div className={`actions-line ${style['left-padding']}`} id={idPrefix}>
+    return (<React.Fragment>
+      <DropdownKebab id='kebab' pullRight className={'visible-xs'}>
```

Comment:
It is unnecessary use brackets here, you can write just: `className='visible-xs'`